dev/packages_try_builders.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
can we use the same paths in all the repos? e.g in the engine repo we are using ci/dev/... additionally let's use just try_builders.json to make the recipes code more generic.

Reply:
I prefer to use the same path as well. 
For engine, as suggested by Yuqian, I saved it to ci/dev.
For sdk, we already have dev. I am not sure if it is okay/good to create a ci from root.
For packages, no dev or ci at all. I guess we can create either way.
For cocoon, it is now in dev, but up to change.

So, it is engine and sdk. what do you think?

Reply:
\cc @goderbauer is it ok to add a new top level folder called ci in this repo? this is to use the same path for builder configurations across repos.

Reply:
Creating a dev folder in this repo sounds good.